app/src/main/java/org/fossasia/openevent/app/common/app/binding/DateBindings.java

```diff
@@ -36,7 +40,12 @@ private static void bindTemporal(Button button, ObservableField<String> date, St
         button.setText(DateUtils.formatDateWithDefault(format, isoDate));
 
         button.setOnClickListener(view -> {
-            ZonedDateTime zonedDateTime = DateUtils.getDate(isoDate);
+            ZonedDateTime zonedDateTime = ZonedDateTime.now();
+            try {
+                zonedDateTime = DateUtils.getDate(isoDate);
+            } catch (DateTimeParseException pe) {
+                Timber.e(pe);
+            }
```

Comment:
Only apply if it is correctly parsed

Reply:
Actually if I do that the date picker dialog won't open when the date is invalid  and then there will be no way of correcting the date.
Done with other requested changes